web/src/topic_popover.js

```diff
@@ -63,6 +63,10 @@ export function initialize() {
                     return;
                 }
 
+                if (instance.context.is_topic_empty) {
+                    return;
+                }
+
                 $popper.on("change", "input[name='sidebar-topic-visibility-select']", (e) => {
                     const start_time = Date.now();
                     const visibility_policy = Number.parseInt(
```

Comment:
It might be a bug to not show the topic visibility feature here; that should be able to operate correctly on the current topic, even if it's empty.